templates/forms/default/form.html.twig

```diff
@@ -16,7 +16,11 @@
     {% endif %}
 {% endfor %}
 
-{% set action = form.action ? base_url ~ form.action : base_url ~ page.route ~ uri.params %}
+{% if form.action %}
+  {% set action = form.action starts with 'http://' or form.action starts with 'https://' ? form.action : base_url ~ form.action %}
```

Comment:
we have a utility to simplify this: `Uri::isExternal($uri)` in Twig this should be accessible by : `grav.uri.isExternal(form.action)` ?

Reply:
not exactly the same, but did it.
That will give expected results, except for form pages starting with `http`